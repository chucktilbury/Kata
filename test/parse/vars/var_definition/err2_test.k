;-------------------------------------------------------------------------------
; verify that invalid var declaration produces error
;-------------------------------------------------------------------------------

var string 123 = 1.0
