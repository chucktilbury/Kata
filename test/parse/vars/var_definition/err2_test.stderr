Syntax: /home/chuck/Src/Kata/test/parse/vars/var_definition/err2_test.k: 5: 15: expected a symbol but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_definition/err2_test.k: 6: 1: expected a variable declaration but got end of file
